release-notes/3.1/3.1.3.md

```diff
@@ -0,0 +1,75 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 3.1.3 released 10 March 2023
+
+This update brings the below changes over the previous release:
+
+### Fixed
+
+- Fixed throttling of token requests by calling AcquireTokenSilent in AAD Integrated/Password flows when the account is already cached.[#1926](https://github.com/dotnet/SqlClient/pull/1926)
+- Fixed TDS RPC error on large queries in SqlCommand.ExecuteReaderAsync.[#1939](https://github.com/dotnet/SqlClient/pull/1939)
+
+## Target Platform Support
+
+- .NET Framework 4.6.1+ (Windows x86, Windows x64)
+- .NET Core 2.1+ (Windows x86, Windows x64, Windows ARM64, Windows ARM, Linux, macOS)
+- .NET Standard 2.0+ (Windows x86, Windows x64, Windows ARM64, Windows ARM, Linux, macOS)
+
+### Dependencies
+
+#### .NET Framework 4.6.1
+
+- Microsoft.Data.SqlClient.SNI 3.0.1
+- Azure.Identity 1.3.0
+- Microsoft.Identity.Client 4.14.0
+- Microsoft.IdentityModel.Protocols.OpenIdConnect 5.6.0
+- Microsoft.IdentityModel.JsonWebTokens 5.6.0
```

Comment:
```suggestion
- Microsoft.Identity.Client 4.22.0
- Microsoft.IdentityModel.Protocols.OpenIdConnect 6.8.0
- Microsoft.IdentityModel.JsonWebTokens 6.8.0
```

Reply:
Looks like there's updates according to the nuget package explorer perhaps from the previous release.